template/Makefile.j2

```diff
@@ -22,8 +22,7 @@ chart-clean:
 	rm -rf deploy/helm/{[ operator.product_string }]-operator/templates/crds.yaml
 
 version:
-	yq eval -i '.version = ${VERSION} | .appVersion = ${VERSION}' deploy/helm/{[ operator.product_string }]-operator/Chart.yaml
-
+	yq -yi '.version = ${VERSION} | .appVersion = ${VERSION}' deploy/helm/zookeeper-operator/Chart.yaml
```

Comment:
The "zookeeper-operator" looks like it might be a mistake here

Reply:
Good point. WIll fix.

Reply:
fa1585968b1ad9bbd165640c3666711a48a99e6d